### DISCUSSION TITLE
Heat up DHW once

### DISCUSSION THREAD

**Discussion by LaMiyu** [edterbak/HeishaMoNR#63] [Ideas]
Hello,

I'm really excited about your solution. Everything is so clearly laid out, clearly structured - I'm a fan :)

If I may express a wish :)

We use a hot water tank - for showering it is enough if it is heated to 43°C every day.
But if my wife wants to take a bath, I have to heat it once to 50°C.

So that my wife can also operate the whole thing, a button would be great:
"Heat once to X°C"

My expectation behind the button would be:
- heat DHW to X °C
- if this is achieved, set DHW to the old value
- Set the mode back to "Heat Only" again

Unfortunately I have no idea how to implement the whole thing myself, apart from writing her instructions on what she has to do ... If it wasn't a big effort, I would welcome it as a feature.

**Comment by edterbak** [comment]
Hi, Thank you for the compliments ofcourse.

I like the idea of pleasing the wife. 1 button/command could be enough.

Question: Does your wife use the dashboard itself? (phone/tablet/pc)
Question 2: Do you have something like an echo or google device for voice command?

I have an Echo dot. In Node Red I use a webhook (https://docs.cb-net.co.uk/en/development/# )
You could program a scene (voice command) like Alexa/hey Google: "I want to take a bath".
The command is sent to google/alexa, the webhook pulls it in Node Red, and you can use that a strigger for a routine..

Just as a nice scenario.

But for this to work, I still need to create a PUSH button to trigger the correct response towards the Panasonic. 
Not difficult to make, but needs to be planned.

**Reply by LaMiyu** [reply]
Hello,

thank you for your answer.

To the 1st question:
Yes, we use a wall tablet that is controlled (Home Assistant) - here I can include the NodeRed dashboard as a separate tile.
So a separate button in the home screen would already really help a lot :)

To the 2nd question:
We also use an Echo Dot - that sounds like a good solution as well. How exactly I integrate Alexa into the Node-Red flows, I'll have to take a closer look.

Many greetings & thanks

**Comment by edterbak** [comment]
2 persons are asking for this functionality now. 
I am working on it

**Comment by edterbak** [comment]
It is implemented in v22 30 and later

**Reply by LaMiyu** [reply]
big thanks :)
